run.py

```diff
@@ -0,0 +1,65 @@
+import os
+from src.model import MLProphet
+from src.data_extract import yahoo
+from sklearn.metrics import r2_score
+from datetime import datetime
+import datetime as dt
+
+
+TICKER = os.environ.get("TICKER", "tsla")
+LABEL = os.environ.get("LABEL", "Open")
+# CURRENT_DATE = os.environ.get("CURRENT_DATE", datetime.today().strftime("%Y-%m-%d"))
+CURRENT_DATE = "2022-04-01"
+WINDOW = os.environ.get("WINDOW", 1)
```

Comment:
should we make window as an optional input from user and with a default value if user doesn't provide?